databases/.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch Tests",
```

Comment:
I think it would be worth waiting to add this launch config until we have tests in the repo. Just to keep dead code out of the main branch